### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (a[target="_blank"][rel="noopener\ noreferrer"])

### ISSUE THREAD

**Issue by jwunderl** [microsoft/pxt-microbit#2294]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://dequeuniversity.com/rules/axe/3.2/color-contrast?application=msftAI))
#### Target application
[Microsoft MakeCode for micro:bit](https://makecode.microbit.org/#editor)
#### Element path
p:nth-child(3) > a[target="_blank"][rel="noopener\ noreferrer"]
#### Snippet
`<a href="https://github.com/lancaster-university/microbit/releases/tag/pxtgc-0" title="microbit runtime version: pxtgc-0" target="_blank" rel="noopener noreferrer">pxtgc-0</a>`
#### How to fix
    Fix any of the following:
      Element has insufficient color contrast of 3.98 (foreground color: #4183c4, background color: #ffffff, font size: 12.0pt, font weight: normal). Expected contrast ratio of 4.5:1
#### Environment
Chrome version 75.0.3770.142
This accessibility issue was found using Accessibility Insights for Web 2.7.1 (axe-core 3.2.2), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

![Screen Shot 2019-07-18 at 2 57 06 PM](https://user-images.githubusercontent.com/5615930/61494924-60605980-a96c-11e9-8a5c-f80b95c766ac.png)
